lib/faraday_middleware/response/caching.rb

```diff
@@ -101,7 +107,8 @@ def cache_on_complete(env)
     end
 
     def store_response_in_cache(key, response)
-      return unless CACHEABLE_STATUS_CODES.include?(response.status)
+      @cacheable_status_code = custom_status_code.any? ? custom_status_code : CACHEABLE_STATUS_CODES
+      return unless @cacheable_status_code.include?(response.status)
```

Comment:
`@cacheable_status_code` is never used outside of this method, so there's no need to make it an instance variable. Please use a local variable instead.

Reply:
suggestion: this logic could be moved into the `cacheable_status_codes` method itself, so there's no need to check with `#any?` on each request 😃